src/Features/Core/Portable/PopulateSwitch/PopulateSwitchExpressionHelpers.cs

```diff
@@ -20,6 +20,12 @@ public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchExpressionOperat
             var switchExpressionType = switchExpression?.Type;
 
             var enumMembers = new Dictionary<long, ISymbol>();
+
+            // Check if the type of the expression is a nullable INamedTypeSymbol
+            // if the type is both nullable and an INamedTypeSymbol extract the type argument from the nullable
+            // and check if it is of enum type
+            switchExpressionType = switchExpressionType.IsNullable(out var underlyingType) ? underlyingType : switchExpressionType;
```

Comment:
The line below was assuming `switchExpressionType` could be null, which is no longer checked here

Reply:
Done